video/out/hwdec/hwdec_drmprime_overlay.c

```diff
@@ -324,12 +335,62 @@ static int init(struct ra_hwdec *hw)
     return -1;
 }
 
+#ifdef AV_HWDEVICE_TYPE_V4L2REQUEST
+static int init_v4l2request(struct ra_hwdec *hw)
+{
+    struct priv *p = hw->priv;
+
+    int ret = pre_init(hw);
+    if (ret != 0)
+        goto err;
+
+    p->hwctx = (struct mp_hwdec_ctx) {
+        .driver_name = hw->driver->name,
+        .hw_imgfmt = IMGFMT_DRMPRIME,
+    };
+
+    /*
+     * AVCodecHWConfig contains a combo of a pixel format and hwdevice type,
+     * correct type must be created here or hwaccel will fail.
+     *
+     * FIXME: Create hwdevice based on type in AVCodecHWConfig
+     */
+    int ret = av_hwdevice_ctx_create(&p->hwctx.av_device_ref,
```

Comment:
video/out/hwdec/hwdec_drmprime_overlay.c:358:9: error: redefinition of ‘ret’ (of line 343)

Reply:
I applied @Kwiboo's suggestions (except from ``meson.options``) and fixed the above problem and then the package build successfully.  
But you need to do ``s/int ret/ret/``